src/KM_Music.pas

```diff
@@ -423,10 +430,12 @@ procedure TMusicLib.UnfadeMusic;
   ZPlayer.GetPosition(StartTime);
   EndTime.ms := StartTime.ms + FADE_TIME;
   ZPlayer.GetPlayerVolume(Left, Right); //Start fade from the current volume
+  Sleep(25); // Sleep for 25 ms, give the stream time to buffer a few bytes
```

Comment:
Ah, so after checking it a bit, no just 25 ms, they are within the {$IFDEF USELIBZPLAY} ... {$ENDIF} and {$IFDEF USEBASS} ... {$ENDIF} tags.

I am going to add a Boolean called fStartState, this will be sel to False at the start and then be set to True when PlayMenuTrack gets called, then the fade will check if the value is True and add the 25 ms delay, in the end it is set to False again so the 25 ms delay will not be used anymore unless it is set to True again.